client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -34,13 +35,42 @@ const SiteTitle: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 	// translators: label for site title input in Gutenboarding
 	const inputLabel = __( 'My site is called' );
 
+	const placeHolder = useTyper(
+		[
+			/* translators: This is an example of a site name,
+			   feel free to create your own but please keep it under 22 characters */
+			_x( 'In Focus Photographers', 'sample site title' ),
```

Comment:
Some of these names are little lengthy and get cut by input's limitations; let's wait a bit for new ones to replace them before deploying.

<img width="379" alt="image" src="https://user-images.githubusercontent.com/87168/84767938-fddc6700-afdb-11ea-878c-e5ae7b97dfb4.png">

Alternatively deploy without the long ones and update the list afterwards if @dubielzyk or @olaolusoga give thumbsup.

Ideally we'd have text fold to 2nd line but it was technically such a hurdle that now is not the right time to tinker with it (https://github.com/Automattic/wp-calypso/issues/43191#issuecomment-643258604).

Feel free to suggest alternatives here! :-)

Reply:
Moar pbAok1-14d-p2